clusters/app.ci/cert-manager/prow_ingress.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: cert-issuer
+  name: prow
+  namespace: ci
+spec:
+  tls:
+    - secretName: prow-tls
+      hosts:
+        - steps.ci.openshift.org
+        - prow.ci.openshift.org
+        - hook.ci.openshift.org
+  rules:
+    - host: steps.ci.openshift.org
```

Comment:
TILd that you can put multiple hosts in one ingress object

Reply:
Is TILd told?
Is it a question, or a remark?

Reply:
"Today I learned"